docs/examples/classification.ipynb

```diff
@@ -0,0 +1,946 @@
+      "text/plain": [
+       "<Figure size 800x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(8, 8))\n",
+    "plot_reliability_diagram(\n",
+    "    y_obs=y_test,\n",
+    "    y_pred=df_pred_test,\n",
```

Comment:
Exclude the trivial model.

Reply:
Done in 66cda548800d51abdae13a160437928d088ed13d